sfs/util.py

```diff
@@ -491,3 +496,26 @@ def _count_walls_1d(a):
         wall_count = wall_count[N_mask, :]
 
     return xs, wall_count
+
+
+def spherical_hn2(n, z):
+    r"""Spherical Hankel function of 2nd kind.
+
+    Defined as http://dlmf.nist.gov/10.47.E6,
+
+    .. math:: \hankel{2}{n}{z} = \sqrt{\frac{\pi}{2z}} \Hankel{2}{n + 1/2}{z},
+
+    where :math:`\Hankel{2}{n}{}` is the Hankel function of the second kind.
+
+    Parameters
+    ----------
+    n : int, array_like
+        Order of the spherical Hankel function (n >= 0).
+    z : complex or float, array_like
+        Argument of the spherical Hankel function.
+
+    Returns
+    -------
+    hn2 : ndarray
```

Comment:
This style is inconsistent with the rest of the file and the toolbox.

Also note that there should be an empty line before the closing `"""`.